scoop/src/main/java/com/lyft/scoop/ViewController.java

```diff
@@ -6,14 +6,32 @@
 
     private boolean attached;
     private Scoop scoop;
+    private View view;
 
-    public void attach(View view) {}
+    public final void attach(View view) {
```

Comment:
We only want the `ViewControllerInflater` calling this correct? In that case make it package protected.